pubtools/_content_gateway/push_staged_cgw.py

```diff
@@ -96,7 +96,7 @@ def push_staged_operations(self):
                 parsed_items = format_cgw_items(parsed_items)
 
                 for pitem in parsed_items:
-                    validate_data(pitem, staged=True)
+                    validate_data(pitem)
```

Comment:
pushItemPath is used at a few places in this function. Could you ensure the absence of it won't break them?

Reply:
Thanks @emilyzheng , fixed.